tests_async/oauth2/test_id_token.py

```diff
@@ -36,6 +36,9 @@ def make_request(status, data=None):
         response.data.read = mock.AsyncMock(
             spec=["__call__"], return_value=json.dumps(data).encode("utf-8")
         )
```

Comment:
Can you remove these lines? They aren't needed anymore.

Reply:
Done